Doc/howto/clinic.rst

```diff
@@ -1137,7 +1140,7 @@ Here's the syntax for cloning a function::
     [clinic start generated code]*/
 
 (The functions can be in different modules or classes.  I wrote
-``module.class`` in the sample just to illustrate that you must
+:py:class:`!module.class` in the sample just to illustrate that you must
```

Comment:
I think that it is rather the AC code than the Python code.

Reply:
After reading it again, I now agree with you. I've reverted the markup change. Thanks for noticing this.

Reply:
See 499ed66666be713b678fa465e24082e92ef61fc5